### ISSUE TITLE
Transfer Nami wallet to Milkomeda Cardano.

### ISSUE THREAD

**Issue by Maxitox84** [Berry-Pool/nami-wallet#490]
Hello. How are they?

Today I have sent 60 ADA from my nami wallet to the Milkomeda Cardano network in metamask. In the detail of the transaction I do not see the metadata or my destination wallet, and I have not received the funds.

Could you explain to me what happened?

The Nami wallet transaction is as follows:
https://cardanoscan.io/transaction/707a986add6cde36861683364a3ef29fe5d6dac5e3f7e94b06b31d8d9392c319

Please i need help thank you

**Comment by alessandrokonrad** [comment]
Hey, not sure how you made the transaction, but the address you sent to is wrong and is not even the milkomeda address.

**Comment by Maxitox84** [comment]
Yes, yesterday I realized that I sent it to a dripdropz wallet, so I will
wait for the refund.
Thank you!

**Comment by CodeStage** [comment]
I just talked to someone who accidentally sent ADA to a BSC address and it ended up in this wallet https://pool.pm/$milkomedaftw - how can that be?!
